app/views/About.js

```diff
@@ -13,9 +13,24 @@ import { useStrategyContent } from '../TracingStrategyContext';
 import packageJson from '../../package.json';
 import { NavigationBarWrapper, Typography } from '../components';
 
+import { useDispatch } from 'react-redux';
+import toggleAllowFeatureFlagsAction from '../store/actions/featureFlags/toggleAllowFeatureFlagsEnabledAction';
+import { TouchableWithoutFeedback } from 'react-native-gesture-handler';
 import { Colors, Spacing, Typography as TypographyStyles } from '../styles';
 
 export const AboutScreen = ({ navigation }) => {
+  const dispatch = useDispatch();
+
+  const [clickCount, setClickCount] = useState(0);
+  useEffect(() => {
+    console.log(clickCount);
+    if (clickCount === 20) {
```

Comment:
what do you think about using a constant for this number?

Also 20 strikes me as a bit annoyingly high of a number, do you think something like 5 would be sufficient?

Reply:
Updated to a constant, good point. 
How about we compromise at 10 😄